### ISSUE TITLE
Odd behavior with file with spaces in its name

### ISSUE THREAD

**Issue by ram535** [bsutton/dshell#43]
When I run this program in the terminal:

```
 okular Introduction\ to\ Recursive\ Programming.pdf
```

It runs normally, but when I run:

```
' okular Introduction\ to\ Recursive\ Programming.pdf'run;
```

It behaves like if I run something like this:

```
okular Introduction
okular to
okular Recursive
okular programming.pdf
```

**Comment by bsutton** [comment]
So the parser in dshell isn't smart enough to handle a file name with
spaces.
Let me have a look at improving the parser.

**Comment by bsutton** [comment]
I've just published 1.0.42 with a fix for this issue.
Instead of using \ just quote the argument.
e.g.

'okular "Introduction to Recursive Programming.pdf"'run;
